Reading List/Reading List/TableView Cell/BookTableViewCell.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  BookTableViewCell.swift
+//  Reading List
+//
+//  Created by Norlan Tibanear on 6/6/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import UIKit
+
+class BookTableViewCell: UITableViewCell {
+    
+    
+    @IBOutlet var titleLabel: UILabel!
+    @IBOutlet var checkButton: UIButton!
+    
+    
+    weak var delegate: BookTableViewCellDelegate?
```

Comment:
Appropriate use of weak here, and like I said nice work figuring out that the delegate needs to be constrained to a class type to make this work.